.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,10 +19,7 @@ Please make sure to fulfil our contribution guideline (https://docs.shopware.com
 ### 4. Please link to the relevant issues (if any).
 
 
-### 5. Which documentation changes (if any) need to be made because of this PR?
-
-
-### 6. Checklist
+### 5. Checklist
 
```

Comment:
we should add a checkbox like: "I written or adjusted the documentation according to my changes" or something link that

Reply:
Good one